src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -20,10 +20,12 @@ const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage })
     <PageBlock
       title={'Work Package Details'}
       headerRight={
-        <>
-          {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
+        <div style={{ display: 'flex', justifyContent: 'flex-end', alignItems: 'center' }}>
```

Comment:
I would be consistent with the rest of codebase by using primarily Grid tags here. You can create a similar design without using style, display, or align items.

Instead, by using a grid and justifying content right